compiler/ir/ir.tree/tree-generator/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+import tasks.WriteCopyrightToFile
+
+plugins {
+    kotlin("jvm")
+    id("jps-compatible")
+    application
+}
+
+val runtimeOnly by configurations
+val compileOnly by configurations
+runtimeOnly.extendsFrom(compileOnly)
+
+dependencies {
+    implementation(project(":generators"))
+    implementation(project(":core:compiler.common"))
+    implementation("com.squareup:kotlinpoet:1.10.1")
+
+    compileOnly(intellijCore())
+    compileOnly(commonDependency("org.jetbrains.intellij.deps:trove4j"))
+
+    runtimeOnly(commonDependency("org.jetbrains.intellij.deps:jdom"))
+}
+
+val writeCopyright by task<WriteCopyrightToFile> {
```

Comment:
It looks unneeded

Reply:
This file is 1-1 copy from FIR btw

Reply:
Yeah, I'd prefer not to touch it unless there's a good reason. Same file in the FIR tree generator has seen some changes related to Gradle configuration caches, and I don't want to dive into it...